packages/bierzo-wallet/src/components/Wallet/ReceiverAddress/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { faQuestionCircle } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import Paper from '@material-ui/core/Paper';
+import TextField from '@material-ui/core/TextField';
+import { makeStyles } from '@material-ui/styles';
+import React from 'react';
+
+interface Props {
+  readonly positionedClass?: string;
+}
+
+const useStyles = makeStyles({
+  toLabel: {
+    outline: '1px solid red',
+    alignSelf: 'flex-start',
+  },
+
+  validityLabel: {
+    outline: '1px solid red',
+    alignSelf: 'flex-start',
+  },
+
+  tooltipContainer: {
+    outline: '1px solid red',
+    display: 'flex',
+    alignSelf: 'flex-end',
+    marginTop: '24px',
+  },
+
+  tooltipLabel: {
+    outline: '1px solid red',
+  },
+
+  tooltipIcon: {
+    outline: '1px solid red',
+    alignSelf: 'center',
+    marginLeft: '10px',
+    color: '#31e6c9',
+  },
+});
+
+const muiClasses = {
+  paper: makeStyles({
+    root: {
+      outline: '1px solid red',
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+      width: '100%',
+      padding: '40px',
+    },
+  }),
+
+  textField: makeStyles({
+    root: {
+      outline: '1px solid red',
+      width: '100%',
+      height: '50px',
+      margin: '16px 0 8px 0',
+    },
+  }),
+};
+
+const ReceiverAddress = ({ positionedClass }: Props) => {
+  const classes = useStyles();
+
+  const overrideClasses = {
```

Comment:
This approach is absolutely different about what we are doing, also I have not seen it anywhere. I recommend to simplify it and take advantage of variable names:
```
const useStyles = makeStyles({
  foo: {
     ...
  },
  bar: {
     ...
  }
});

const ReceiverAddress = (...) => {
    const classes = useStyles();
    const paperClasses = {
       root: classes.foo,
   }
   const textFieldClasses = {
       root: classes.bar
   }

  ....
``` 

Reply:
Please, update all components and follow this approach. In this way, with the variable name,  you can get a direct hint about what is going on `textFieldClasses --> classes for textField` 😉.

Reply:
Fixed in [7b42812](https://github.com/iov-one/ponferrada/pull/79/commits/7b4281279aca898a48cac504a7f98a8e7cbbf32b).